tables/Tables/ArrayColumn.tcc

```diff
@@ -194,17 +194,17 @@ Array<T> ArrayColumn<T>::getSlice (uInt rownr,
 template<class T>
 void ArrayColumn<T>::getSlice (uInt rownr, const Slicer& arraySection,
                                Array<T>& arr, Bool resize) const
-{
+                               {
```

Comment:
I don't know why it got out there.  Sometimes the editor gets confused 
and does that.  When I see it I put it back where it belongs.  Sorry.

On 06/25/2015 12:31 AM, Ger van Diepen wrote:
>
> In tables/Tables/ArrayColumn.tcc 
> <https://github.com/casacore/casacore/pull/121#discussion_r33225769>:
>
> > @@ -194,17 +194,17 @@ Array<T> ArrayColumn<T>::getSlice (uInt rownr,
> >  template<class T>
> >  void ArrayColumn<T>::getSlice (uInt rownr, const Slicer& arraySection,
> >                                 Array<T>& arr, Bool resize) const
> > -{
> > +                               {
>
> Why indented so far? Is it done by your editor or another tool?
> In principle current style should be kept.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/casacore/casacore/pull/121/files#r33225769>.Web 
> Bug from 
> https://github.com/notifications/beacon/AGPGtJ4kAmbmkUA0X-xV8t7nCS2CJjJ4ks5oW5e-gaJpZM4FDwg4.gif
>

-- 
Jim Jacobs

phone: 835-7235
office: 301